cli/generator/generate.js

```diff
@@ -0,0 +1,54 @@
+const chalk = require('chalk')
+const cliInput = require('./cliInput')
+const RootDirController = require('./RootDirController')
+const packageMerge = require('./packageMerge')
+const exec = require('child-process-promise').exec
+const depcheck = require('depcheck')
+const path = require("path")
+
+module.exports = {
+  run: async () => {
+    const projectDir = cliInput.get('projectDirectory')
+    const rootDir = new RootDirController(projectDir)
+    let peerDepInstall
+    // Create directory for new implementing application
+    if (!rootDir.has()) rootDir.make()
+
+    return new Promise( async (resolve, reject) => {
+      // Clone ol-kit directory into new implementing application
+      console.log(chalk.cyan(`    Cloning ol-kit files into /${projectDir}`))
+      return exec(`git clone https://github.com/MonsantoCo/ol-kit.git ./${projectDir}`) 
+      .then(() => {
+        // If our peer dependencies ship a breaking change this will need to be addressed.
+        const peerDependencies = require(`${rootDir.projectPath}/package.json`).peerDependencies
+        peerDepInstall = Object.keys(peerDependencies).join(' ')
```

Comment:
Find package to read versions correctly OR regex to install latest major

Reply:
This is what NPM itself uses (to my recollection) https://www.npmjs.com/package/read-package-json

Reply:
See the newest changes, we are adding a script to only get the latest major releases. E.g. if its `v8.2.1` we only want `8`